buildstockbatch/gcp/gcp.py

```diff
@@ -1226,7 +1235,7 @@ def main():
             logger.setLevel(logging.INFO)
 
         # validate the project, and if --validateonly flag is set, return True if validation passes
-        GcpBatch.validate_project(os.path.abspath(args.project_filename), args.missingonly)
+        GcpBatch.validate_project(os.path.abspath(args.project_filename))
```

Comment:
should we also add `args.missingonly` here and avoid checking for that in the check_output_dir?

Reply:
I'm not sure if this is what you meant, but I moved the `missingonly` check out of `check_output_dir` and put it below.